css/styles.css

```diff
@@ -1,73 +1,93 @@
-* {
-	box-sizing: border-box;
-}
-
-body {
-	background-color: #fff;
-	margin: 0 auto;
-	text-align: center;
-}
-
-a {
-	text-decoration: none;
-	color: rgba(71, 71, 107, .7);
-}
-
-
-/* header and navigation menu */
-
-.minimo {
-	padding-top: 30px;
-	padding-bottom: 20px;
-}
-
-.main-nav ul {
-	margin: 0 auto;
-}
-
-.main-nav li {
-	font-family: "Playfair Display", serif;
-	color: rgba(71, 71, 107, .7);
-	font-size: 0.8rem;
-	font-weight: 500;
-	text-transform: uppercase;
-	padding: 5px 0;
-}
-
-
-@media (min-width: 768px) {
-	
-	.minimo {
-		float: left;
-		padding-left: 40px;
-		padding-top: 40px;
-	}
-	
-	.main-nav ul {
-		float: right;
-		padding-top: 35px;
-		padding-right: 15px;
-	}
-	
-	.main-nav li {
-		display: inline-block;
-		padding: 5px 15px;
-		font-size: 0.9rem;
-	}
-}
-
-@media (min-width: 1024px) {
-	.minimo {
-		padding-left: 80px;
-	}
-	
-	.main-nav ul {
-		padding-right: 25px;
-	}
-	
-	.main-nav li {
-		padding: 5px 55px;
-		font-size: 1.1rem;
-	}
-}
-
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background-color: #fff;
+}
+
+a {
+	text-decoration: none;
+	color: rgba(71, 71, 107, .7);
+}
+
+.container {
+	max-width: 1200px;
+	margin: 0 auto;
+	text-align: center;
+}
+
+/* header and navigation menu */
+
+.minimo {
+	padding-top: 30px;
+	padding-bottom: 20px;
+}
+
+.main-nav ul {
+	margin: 0 auto;
+}
+
+.main-nav li {
+	font-family: "Playfair Display", serif;
+	color: rgba(71, 71, 107, .7);
+	font-size: 0.8rem;
+	font-weight: 500;
+	text-transform: uppercase;
+	padding: 5px 0;
+}
+
+.header-photo {
+	background-image: url("../images/Header_photo.jpg");
+	background-repeat: no-repeat;
+	background-position: center;
+	height: 600px;
+}
+
+.header-photo img {
+	margin: 0 auto;
+	width: 90%;
+}
+
+.clearfix:after {
+	content: "";
+	display: table;
+	clear: both;
+}
+
+@media (min-width: 768px) {
+	
+	.minimo {
+		float: left;
+		padding-left: 40px;

```

Comment:
Nie podobają mi się te boczne paddingi tutaj. To troszkę jak tetris teraz jest i w którymś momencie Ci się to sypnie. Już widzę, że zdjęcie w headerze ma mniejszą szerokość niż box, w którym się znajduje, bo jakby było na 100% szerokości to by wystawało poza kontent. Lepiej zrobić wrapper, dookoła wszystkich elementów, który będzie miał boczne paddingi. Coś w tym stylu: https://jsfiddle.net/norin89/1s46dad1/2/ - dorzuć u siebie po bannerem jakiś paragraf i zobacz jak ucieka w lewo. Dzięki takiemu wrapperowi, łatwiej później zarządzać kodem.

EDIT. Widzę, że masz u siebie `.container`. Proponuję zatem żebyś:
* wyrzucił lewe paddingi z `.minimio` i prawy z `.main-nav ul` (dla wszystkich breakpointów)
* dorzucił `.main-nav li:last-child { padding-left: 0; }` by dociągnąć ostatni element menu do prawej krawędzi kontenera,
* dał `padding: 0 40px;` dla `.container`.
* dorzucił `background-size: cover;` do obrazka, by ten zajął całą szerokość elementu w którym jest - dzięki temu będziesz widział, że wszystko trzyma się jednej "ramy", którą jest `.container`. Teraz zamiast dla różnych breakopintów edytować paddingi dla `.main-nav ul`, `.main-nav li`, `.minimo`, itp.. zmieniasz tylko paddingi i jeśli potrzebujesz to też szerokość dla `.container`.